Gen3UI/Gen3UI.pro

```diff
@@ -12,7 +12,7 @@ TARGET = Gen3UI
 TEMPLATE = lib
 
 CONFIG += ffmpeg
-#CONFIG  += wince  ##wince
+CONFIG  += wince  ##wince
```

Comment:
I am not sure set the WinCE port as default port is necessary, maybe mentioned this in the instruction doc is better.